tests/console/zypper_ar.pm

```diff
@@ -1,29 +1,41 @@
 # SUSE's openQA tests
 #
 # Copyright © 2009-2013 Bernhard M. Wiedemann
-# Copyright © 2012-2016 SUSE LLC
+# Copyright © 2012-2018 SUSE LLC
 #
 # Copying and distribution of this file, with or without modification,
 # are permitted in any medium without royalty provided the copyright
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# G-Summary: harmorize zypper_ref between SLE and openSUSE
-# G-Maintainer: Max Lin <mlin@suse.com>
+# Summary: harmorize zypper_ref between SLE and openSUSE
+# Maintainer: Max Lin <mlin@suse.com>
```

Comment:
Not a good summary and also we should ask of maxlin will maintain.

@nilxam will you?

Reply:
as this test is only be used in openSUSE, so fine with me, I can take care of them